index.bs

```diff
@@ -1107,13 +1110,37 @@ To <dfn>attempt to deliver a report</dfn> given an [=attribution report=] |repor
     ::  "`omit`"
     :   `cache mode`
     ::  "`no-store`"
+1. Return |request|.
+
+<h3 id="issue-report-request">Issuing a report request</h3>
+
+This algorithm constructs a [=request=] and attempts to deliver it to
+|report|'s [=attribution report/reporting endpoint=].
+
+To <dfn>attempt to deliver a report</dfn> given an [=attribution report=] |report|, run the following steps:
+
+1. Let |url| be the result of executing [=generate a report URL=] on |report|.
+1. Let |request| be the result of executing [=create a report request=] on |report|.
 1. [=Queue a task=] to [=fetch=] |request| with [=fetch/processResponse=] being these steps:
     1. [=Queue a task=] to remove |report| from the [=attribution report cache=].
 
 Issue(220): This fetch should use a network partition key for an opaque origin.
 
 A user agent MAY retry this algorithm in the event that there was an error.
 
+<h3 id="issue-debug-report-request">Issuing a debug report request</h3>
+
+To <dfn>attempt to deliver a debug report</dfn> given an [=attribution report=] |report|:
+
+1. Let |url| be the result of executing [=generate a report URL=] on |report| with
+    [=generate a report URL/isDebugReport=] set to true.
+1. Let |request| be the result of executing [=create a report request=] on |report|.
+1. [=Fetch=] |request|.
```

Comment:
I think we still want to [=Queue a task=] to fetch request, we want it to be async.

Reply:
But this algorithm was already called from a queued task.

Reply:
Totally missed that, please ignore. (It'd be nice if there was someway to mark an algorithm async and require it be called in that manner)